extensions/amp-ad-network-doubleclick-impl/amp-ad-network-doubleclick-impl-internal.md

```diff
@@ -72,123 +90,81 @@ Below the term `primary size` refers to the width and height pair specified by t
-is idle.  The result is an increase in impressions with a much smaller increase in
-viewable impressions and clicks.  Publishers sensitive to viewability rate should
-set data-loading-strategy=3 to keep the current viewport offset and disable idle render.  Publishers using data-loading-strategy=prefer-viewability-over-views will
-use current 1.25 viewport offset with idle render disabled.
-
-### AMP Ad Refresh
+### Supported parameters
 
-AMP Ad Refresh permits amp-ad tags using Fast Fetch to undergo periodic refresh events. Each such event re-issues a new ad request and attempts to display the returned creative.
+- `data-slot`
+- `data-multi-size`
+- `data-multi-size-validation`
 
-#### Network-level Configuration
+Supported via `json` attribute:
 
-For a network to make use of Refresh, its corresponding implementation must create an instance of RefreshManager and call
-`initiateRefreshCycle`. The simplest way to do so is to use the exported `getRefreshManager` function from refresh-manager.js:
+- `categoryExclusions`
+- `cookieOptions`
+- `tagForChildDirectedTreatment`
+- `targeting`
+- `useSameDomainRenderingUntilDeprecated`
 
-```javascript
-import {getRefreshManager} from '../../amp-a4a/0.1/refresh-manager';
+### Unsupported DFP Formats
+- Interstitials
+- Expandables. Although expandables on interaction/click is a format that is work in progress.
+- Flash
+- Anchor Ads / Adhesion Units
+- Creatives served over HTTP.
 
-// ...
-
-const refreshManager = getRefreshManager(this);
-refreshManager.initiateRefreshCycle();
-```
-
-While `getRefreshManager` is convenient, it does not allow for customization. It returns a RefreshManager with default configurations (specified in the table below). To customize the RefreshManager, you must invoke its constructor directly:
-
-```javascript
-import {
-  getRefreshManager,
-  getPublisherSpecifiedRefreshInterval,
-} from '../../amp-a4a/0.1/refresh-manager';
-
-// ...
-
-const refreshConfig = { /* see table below for configuration parameters */ };
-const refreshInterval = getPublisherSpecifiedRefreshInterval(this.element, this.win);
-const refreshManager = new RefreshManager(this, refreshConfig, refreshInterval);
-refreshManager.initiateRefreshCycle();
-```
-
-Optionally, a network may override AmpA4a's `refresh` method, which would allow it to insert custom logic in between refresh events. Note: RefreshManager passes a callback to `refresh` which must be invoked in order to reset the cycle; if it is not invoked, the RefreshManager will become idle until either the callback or `initiateRefreshCycle` is called. If `refresh` is not overridden, this is handled automatically by AmpA4a.
+### Supported Features
 
 <table>
   <tr>
-    <td>Parameter</td>
-    <td>Description</td>
-    <td>Permitted Values</td>
-    <td>Default Value</td>
+    <td><strong>Feature</strong></td>
```

Comment:
Can we move this up so its just below the first table?  Can we also populate the Examples in that first table which currently says TBD

Reply:
Table moved. Seems a bit redundant, but I added a link to the example section below.